src/components/schemas/employee.yaml

```diff
@@ -203,6 +203,21 @@ employee_out:
         $ref: ./fields.yaml#/delegatees
       description: |
         List of users who are delegatees for this employee.
+    category_ids:
+      type: array
+      nullable: False
+      items:
+        $ref: './fields.yaml#/id_integer'
+      example: [ 3345, 3346 ]
+      description: |
```

Comment:
<img width="679" alt="Screenshot 2023-08-22 at 3 23 12 PM" src="https://github.com/fylein/fyle-platform-docs/assets/22430409/7bccbede-ec1c-40fb-99e0-acb4fac5128b">

This is a nullable key right @satyamyesj ? 

Do we throw error when user sends category_ids but the employee category restriction is disabled ?

Reply:
@modernwarfareuplink updated API contract, this field should not be null. No we are no doing any validation depending upon category restriction is disabled or not.

Reply:
@satyamyesj If it is already decided then it should be fine. But all other fields in the employees invite api uses the empty array/null approach but it is different for the categories.

<img width="592" alt="Screenshot 2023-08-22 at 3 52 04 PM" src="https://github.com/fylein/fyle-platform-docs/assets/22430409/49d52ef1-78e0-4d8b-8ad0-bf1810bc84bb">

Reply:
@modernwarfareuplink I don't undestand this, in DB we have marked project_ids, cost_center_ids as not null fields then on GET API how we are going to return null ?

Reply:
Seems like we don't accept `null` values for other fields and this is an error in doc.

Reply:
@modernwarfareuplink Updated docs for other fields as well.